qt-models/diveimportedmodel.cpp

```diff
@@ -136,10 +136,14 @@ void DiveImportedModel::repopulate(dive_table_t *table, struct dive_site_table *
 	diveTable = table;
 	sitesTable = sites;
 	firstIndex = 0;
-	lastIndex = diveTable->nr - 1;
-	checkStates.resize(diveTable->nr);
-	std::fill(checkStates.begin(), checkStates.end(), true);
-
+	if (diveTable) {
```

Comment:
How can table be null here? Even if this doesnt crash, import still will not work, isnt it?

Reply:
I'm frankly as perplexed as you are... but this was the only thing that I thought could be crashing in that function with the stack trace in #2221